packaging/windows/doinstaller_wix.cmd

```diff
@@ -119,7 +119,7 @@ IF %ERRORLEVEL% NEQ 0 (
 :: Run keybase sign to get signature of update
 set KeybaseBin="%LOCALAPPDATA%\Keybase\keybase.exe"
 set SigFile=sig.txt
-%KeybaseBin% sign -d -i %KEYBASE_INSTALLER_NAME% -o %SigFile%
+%KeybaseBin% sign -d --saltpack-version=1 -i %KEYBASE_INSTALLER_NAME% -o %SigFile%
```

Comment:
I'd prefer making all these arguments use their `--full-names` for readability, but that's a preexisting condition :)